app/templates/eventNav.html

```diff
@@ -1,12 +1,12 @@
 {% extends "base.html"%}
 
-{% macro eventheader(page_title, eventData, size, isNewEvent) %}
+{% macro eventheader(page_title, event, size, isNewEvent) %}
     {% set bigh = 'h1' if size == 'large' else 'h3' %}
     {% set smallh = 'h5' if size == 'large' else 'p' %}
 
     <{{bigh}} class="mb-0" id="title">{{page_title}}</{{bigh}}>
     {% if not isNewEvent %}
-        {% set programName = "CELTS-Sponsored" if eventData["program"] == None else eventData["program"].programName %}
+        {% set programName = "CELTS-Sponsored" if event.program == None else event.program.programName %}
```

Comment:
This shouldn't be needed because there should never be an event without a program.